packages/devtools_app/lib/src/service/service_extensions.dart

```diff
@@ -445,7 +444,7 @@ final toggleOnDeviceWidgetInspector =
       // toggling widget select mode as it is the only way to toggle that mode.
       title: 'Select Widget Mode',
       shortTitle: 'Select',
-      iconData: CupertinoIcons.cursor_rays,
+      iconData: const IconData(0x1F74A),
```

Comment:
can you put this in a named const so we can add dart doc to make it clear why we are using this icon and that we shouldn't change it?